.circleci/config.yml

```diff
@@ -90,6 +81,26 @@ jobs:
       DO_ENV: py27
 
 
+  test_premium:
+    docker:
+      - image: circleci/python:2.7
+
+    steps:
+      - checkout
+      - run: sudo apt-get install libldap-dev libsasl2-dev
+      - run: pip install --user tox
+      - run:
+          name: Avoid hosts unknown for github
+          command: mkdir -p ~/.ssh && echo -e "Host github.com\n\tStrictHostKeyChecking no\n" >> ~/.ssh/config
+
+      - run: git clone git@github.com:cloudify-cosmo/cloudify-premium.git ~/cloudify-premium
+      - run: |
+          cd ~/cloudify-premium
+          git checkout "${CIRCLE_BRANCH}" || echo no matching branch found, using master
+      - run: sed -i "s:^.*cloudify-manager.*\(rest-service\)$:-e ${HOME}/project/\1/:" ~/cloudify-premium/dev-requirements.txt
```

Comment:
Maybe add a comment here on what this does?

Reply:
Added